docs/topics/roadmap.md

```diff
@@ -159,8 +162,9 @@ Visit the [new roadmap board in our issue tracker ![YouTrack](youtrack-logo.png)
         <td>
             <ul>
                 <li><a href="https://youtrack.jetbrains.com/issue/KT-42304" target="_blank">Improve IDE performance</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/TODO" target="_blank">Refactor the internal index system</a></li>
                 <li><a href="https://youtrack.jetbrains.com/issue/KT-44330" target="_blank">Improve debugging experience</a></li>
-                <li><a href="https://youtrack.jetbrains.com/issue/KT-42306" target="_blank">Improve cross-language support in the Inline Method and Change Signature refactorings</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/TODO" target="_blank">Improve the New Project wizard</a></li>
```

Comment:
Who is responsible for creating tickets for this and "Refactor the internal index system" items?
Should the tickets be already created?

Reply:
@specherkin has created all the new tickets in YT, I've added links accordingly.